lib/helpers.js

```diff
@@ -122,3 +127,15 @@ export const setupModule = targetComponent => module => {
   }
   return moduleTemplate;
 };
+
+export function checkExistence(name, list = [], local = false) {
```

Comment:
This function feels a bit weird to me due to it being polymorphic, it seems like its designed to be in a `try/catch` where it would throw the error message in the object?